sdk/table/azure-table/README.md

```diff
@@ -0,0 +1,354 @@
+    this client represents interaction with a specific table (which need not exist yet). It provides operations to
+    create, delete, or update a table and includes operations to query, get, and upsert entities
+    within it.
+    
+### Entities
+* **Create** - Adds an entity to the table.
+* **Delete** - Deletes an entity from the table.
+* **Update** - Updates an entities information by either merging or replacing the existing entity.
+* **Query** - Queries existing entities in a table based off of the QueryOptions (OData).
+* **Get** - Gets a specific entity from a table by partition and row key.
+* **Upsert** - Merges or replaces an entity in a table, or if the entity does not exist, inserts the entity. 
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Table tasks, including:
+
+* [Creating a table](#creating-a-table "Creating a table")
+* [Creating entities](#creating-entities "Creating entities")
+* [Querying entities](#querying-entities "Querying entities")
+
+
+### Creating a table
+Create a table in your storage account
+
+```python
+from azure.table import TableServiceClient
+
+table = TableServiceClient.from_connection_string(conn_str="<connection_string>")
+table.create_table()
+```
+
+Use the async client to create a table
+```python
+from azure.table.aio import TableServiceClient
+
+table = TableServiceClient.from_connection_string(conn_str="<connection_string>")
+await table.create_table()
+```
+
+### Creating entities
+Create entities for table
+
+```python
+from azure.table import TableClient
+
+my_entity = {'PartitionKey':'part','RowKey':'row'}
+
+table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="my_table")
+entity = table.create_entity(table_entity_properties=my_entity)
+```
+
+Create entities asynchronously
+
+```python
+import asyncio
+from azure.table.aio import TableClient
+
+my_entity = {'PartitionKey':'part','RowKey':'row'}
+
+table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="my_table")
+entity = await table.create_entity(table_entity_properties=my_entity)
+```
+
+### Querying entities
+Querying entities for table
+
+```python
+from azure.table import TableClient
+from azure.table._generated.models import QueryOptions
+
+query = QueryOptions(top=3)
+
+table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="my_table")
+entity = table.query_entities(query_options=query)
+```
+
+Querying entities asynchronously
+
+```python
+import asyncio
+from azure.table.aio import TableClient
+from azure.table._generated.models import QueryOptions
+
+query = QueryOptions(top=3)
+
+table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="my_table")
+entity = await table.query_entities(query_options=query)
+```
+
+## Optional Configuration
+
+Optional keyword arguments that can be passed in at the client and per-operation level. 
```

Comment:
Other libraries are just linking out to the azure-core docs instead of adding all the arguments here. Take a look at this:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/README.md#optional-configuration